src/ParallelAlgorithms/Interpolation/Protocols/InterpolationTargetTag.hpp

```diff
@@ -0,0 +1,119 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+#include <type_traits>
+
+#include "ParallelAlgorithms/Interpolation/InterpolationTargetDetail.hpp"
+#include "ParallelAlgorithms/Interpolation/Protocols/ComputeTargetPoints.hpp"
+#include "ParallelAlgorithms/Interpolation/Protocols/ComputeVarsToInterpolate.hpp"
+#include "ParallelAlgorithms/Interpolation/Protocols/PostInterpolationCallback.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+
+namespace intrp::protocols {
+namespace detail {
+// Both will have `::value = true` because we want the static assert in the
```

Comment:
Unclear what 'Both' means.  Maybe "if_alias_exists_assert_conforms_to::value will always be true because..."